### ISSUE TITLE
v1.14.1: NullReferenceException w/PROXY_FOR_REMOTE_ADAPTER

### ISSUE THREAD

**Issue by GeorgeTsiokos** [Lightstreamer/Lightstreamer-lib-adapter-dotnet-remote#1]
After upgrade to `1.14.1`, I see this exception reported in the .net data adapter:
```
      System.NullReferenceException: Object reference not set to an instance of an object.
         at Lightstreamer.DotNet.Server.RemotingProtocol.DecodeStringOld(String str)
         at Lightstreamer.DotNet.Server.RemotingProtocol.EncodeString(String str)
         at Lightstreamer.DotNet.Server.DataProviderProtocol.WriteUpdateByEvent(String itemName, String requestID, IItemEvent itemEvent, Boolean isSnapshot)
         at Lightstreamer.DotNet.Server.DataProviderServerImpl.Update(String itemName, IItemEvent itemEvent, Boolean isSnapshot)
```
Downgrading to `1.13.1` resolved this issue. It appears the null ref check on `str` was removed:

![image](https://user-images.githubusercontent.com/1034086/201357346-ab348d44-fdfa-4d57-a8d6-8fd94c671f97.png)

**Comment by Bravo27** [comment]
Hi George,

Thank you for reporting the issue.
We will check and eventually provide for a release of the fix.

**Comment by Bravo27** [comment]
Fixed with commit [be1350c](https://github.com/Lightstreamer/Lightstreamer-lib-adapter-dotnet-remote/commit/be1350ce73e501356a5ee1ce81587b69ec5634cf).
